### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.  Just another depricated option.

### ISSUE THREAD

**Issue by barabadzhi** [joekr/theme-toggler#2]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.

```
Atom.Object.defineProperty.get (/opt/atom/resources/app/src/atom.js:53:11)
Object.activate (/home/bogdan/.atom/packages/theme-toggler/lib/theme-toggler.coffee:17:9)
```

:small_red_triangle: Here is an error output.

```
ThemeTogglerView.initialize (/home/bogdan/.atom/packages/theme-toggler/lib/theme-toggler-view.coffee:16:9)
```

:small_red_triangle: Here is another one/same issue :coffee:

**Comment by joekr** [comment]
Yeah, I noticed this the other day. Thanks for making the issue :+1:

An update will have to be made before these deprecated calls are removed completely. Lucky they are warnings now and not breaking things. :sweat:

**Comment by joekr** [comment]
Thanks for the help :+1:. The fix will be pushed as part of v0.3.4 later this week.
